src/Engine.ts

```diff
@@ -144,6 +147,11 @@ export class Engine {
         });
         Handlebars.registerHelper("toPropertyName", str => {
             let retStr = "";
+
+            if (this.Options.removeIdSuffix && str.length > 3 && str.endsWith("_id")) {
```

Comment:
It's popular to use "id" without underscore. I think it would be good to check for both.

Reply:
However I'm not sure if it should be implemented here. We shouldn't change names which are mapped to database columns, only those pointing to related objects.
Probably better place is in src/Engine.ts:28 after getting data from a server.